tuning/utils/preprocessing_utils.py

```diff
@@ -65,16 +59,32 @@ def validate_data_args(data_args: configs.DataArguments, packing: bool):
                 "dataset_text_field and data_formatter_template are both set,\
                 but are mutually exclusive options"
             )
-    # TODO(s) In future seupport two more formats:
-    # 1. Allow no response template, and JSON with input/output fields and mask input
 
-    # 2. Allow pretokenized Dataset besides JSON.
+    # If not single sequence, JSON should contain input/output fields
+    if not (data_args.dataset_text_field or data_args.data_formatter_template):
+        json_dataset = datasets.load_dataset(
+            "json", data_files=data_args.training_data_path
+        )
+        if "input" not in json_dataset["train"].column_names:
```

Comment:
Can you add two const vars 
```python
INPUT_KEY = "input"
OUTPUT_KEY = "output"
```
at the top of this file and check against the variables instead of hardcoding the value here?

Then it can also be used in the calls to `get_preprocessed_dataset` and the error messages below and only be hardcoded in one place